### DISCUSSION TITLE
Timeseries analysis using GE

### DISCUSSION THREAD

**Discussion by bluemelon12** [great-expectations/great_expectations#5064] [Support] [answered]
Hi,
I want to use GE to compare current data points with data points 1 month back and see if change is less than threshold percent. How do I implement it using GE

**Comment by austiezr** [comment, marked as answer]
Hey @bluemelon12 ! Thanks for reaching out. While we do have eyes on this functionality, Great Expectations does not currently support native historical metric comparison. However, the results of every validation are stored as a json object in the `great_expectations/uncommitted/validations/<expectation_suite_name>/<run_id>/<timestamp>` directories of your project. You could then compare results of runs yourself over time.
